### PR TITLE
*: remove explicit health check

### PR DESCRIPTION
**Pull request by hasbro17**
ref: #1673

### CODE REVIEW

**Comment by hongchaodeng** [comment]
```
hack/test: line 159: e2esh_pass: command not found
```
Rebase?

**Comment by hasbro17** [comment]
@hongchaodeng @fanminshi PTAL

**Comment by fanminshi** [comment]
lgtm!

File: pkg/cluster/cluster.go [hunk 1]
```diff
@@ -443,24 +443,26 @@ func (c *Cluster) pollPods() (running, pending []*v1.Pod, err error) {
...
+		if !presentIn(m.Name, running) {
 			unready = append(unready, m.Name)
 		}
 	}
-	c.status.Members.Ready = ready
+	c.status.Members.Ready = running
 	c.status.Members.Unready = unready
 }
 
+func presentIn(a string, list []string) bool {
```
Comment (hongchaodeng) [inline pkg/cluster/cluster.go]: duplicate to presentIn() in e2euitl/wait_util.go

**Comment by hasbro17** [comment]
Fixed. Moved `presentIn` to `pkg/util/util.go`.

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**